syntax = "proto3";

package log;

service Logger { rpc WriteLog(LogEntry) returns (LogResponse); }

message LogEntry {
  int32 term = 1;
  int32 index = 2;
  string command = 3;
  string data = 4;
}

message LogResponse {
  bool ack = 1;
  int32 curr_term = 2;
  int32 next_index = 3;
}
